--[[ 
  - ENGO EXPLORER -
  v0.3.* offical loader
  drawing library based explorer

  Copy this loadstring for internet load via engos.site (slower).
  loadstring(game:HttpGet('https://engos.site/explorer.luau'))()
  
  Copy this loadstring for internet load via github repository (slow)
  loadstring(game:HttpGet('https://raw.githubusercontent.com/joeengo/engo-explorer/refs/heads/main/init.lua'))()

  Copy this loadfile for full file load (fastest) - note: run internet loadstring before using files for now
  loadfile('explorer-v3/init.lua')()

  engo explorer is still a work in progress - this is a beta release
]]


-- Default loadstring
loadstring(game:HttpGet('https://raw.githubusercontent.com/joeengo/engo-explorer/refs/heads/main/init.lua'))()
